scaladoc/README.md

```diff
@@ -63,10 +63,17 @@ You can also find the result of building the same sites for latest `master` at:
 - <https://scala3doc.virtuslab.com/master/scala3/main/index.html>
 - <https://scala3doc.virtuslab.com/master/testcases/main/index.html>
 
+You can run the Scaladoc tool from the sbt prompt as a user would run it
+with `scaladoc/run`, passing `-d <directory>` and some `.tasty` files as
+input on the command line.
+
 ### Testing
 
-Most tests rely on comparing signatures (of classes, methods, objects etc.) extracted from the generated documentation
-to signatures found in source files. Such tests are defined using [SignatureTest](test/dotty/tools/scaladoc/signatures/SignatureTest.scala) class
+From the sbt prompt, `scaladoc/test` runs the tests.
+
+Many tests simply consist of small Scala source files to be documented. In this style of test, it's not necessary for expected output to be included, because the test is its own specification. The testing infrastructure compares the signatures (of classes, methods, objects etc.) extracted from the generated documentation to the signatures extracted from the same source file using Scalameta.
+
+Such tests are defined using [SignatureTest](test/dotty/tools/scaladoc/signatures/SignatureTest.scala) class
 and its subtypes (such as [TranslatableSignaturesTestCases](test/dotty/tools/scaladoc/signatures/TranslatableSignaturesTestCases.scala))
```

Comment:
I am not sure if its useful but maybe you can include that the source files are actually present in the `scaladoc-testcases` subproject in `src/tests`

Reply:
done